di_website/datasection/blocks.py

```diff
@@ -0,0 +1,54 @@
+from wagtail.core.blocks import (
+    PageChooserBlock,
+    RichTextBlock,
+    StreamBlock,
+    StructBlock,
+    TextBlock,
+    URLBlock,
+    EmailBlock
+)
+from wagtail.core.fields import StreamField
+from wagtail.images.blocks import ImageChooserBlock
+
+
+class TeamMemberQuoteBlock(StructBlock):
+    quote_text = TextBlock()
+    team_member = PageChooserBlock(page_type='ourteam.TeamMemberPage', required=False)
+
+    class Meta():
+        icon = 'fa-users'
+
+
+class ExternalMemberQuoteBlock(StructBlock):
+    quote_text = TextBlock()
+    name = TextBlock()
+    role = TextBlock()
+    organisation = TextBlock()
+    external_member_photo = ImageChooserBlock(required=False)
+
+    class Meta():
+        icon = 'fa-user'
+
+
+class QuoteStreamBlock(StreamBlock):
+    team_member = TeamMemberQuoteBlock()
+    external_member = ExternalMemberQuoteBlock()
+    required = False
+
+
+class DataSupportServicesBlock(StructBlock):
+    title = TextBlock()
+    description = TextBlock()
```

Comment:
Hi David. Had a chance to re-review. All of these look good except for most of them could you please add `required=False`? I think description, telephone, email, photographer, and photo credit all need `required=False`. Your template already has booleans for most of them so they don't appear if none, so users just need the ability to omit sections.

Reply:
Made all these fields optional